.github/dependabot.yaml

```diff
@@ -0,0 +1,12 @@
+version: 2
+updates:
+  # See documentation for possible values
+  # > "https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuration-options-for-dependency-updates"
+  - package-ecosystem: "gomod"
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "daily"
+      time: "12:00"
```

Comment:
Personally I would prefer an earlier time so we can see these in the morning and they don't get lost in the PR shuffle, but that's just my personal preference

Reply:
Good call.